onediff_diffusers_extensions/examples/sd35/README.md

```
@@ -0,0 +1,145 @@
+# Ensure diffusers include the SD35 pipeline.
+pip3 install --upgrade diffusers[torch]
+```
+### Set up SD35
+Model version for diffusers: https://huggingface.co/stabilityai/stable-diffusion-3.5-large
+
+HF pipeline: https://huggingface.co/stabilityai/stable-diffusion-3.5-large/blob/main/README.md
+
+## Run
+
+### Run 1024*1024 without compile (the original pytorch HF diffusers baseline)
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)